docs/running-rules.md

```diff
@@ -9,42 +9,33 @@ import MoreHelp from "/src/components/MoreHelp"
 
-- Use existing [Semgrep Registry rules](#running-semgrep-registry-rules) or add new rules to Semgrep Registry and run them.
+- Existing [Semgrep Registry rules](#running-semgrep-registry-rules). See [Contributing to Semgrep rules](https://semgrep.dev/docs/contributing/contributing-to-semgrep-rules-repository/) to find out how to add new rules.
 - Local rules:
-  - Create and run [ephemeral rules](#ephemeral-rules) in the command line.
-  - Create and run [YAML-defined rules](#creating-and-using-yaml-defined-rules-file).
-- Run your [local rules simultaneously with Semgrep Registry rules](#running-multiple-rules-simultaneously). 
+  - One-off [ephemeral rules](#ephemeral-rules) used in the command line.
+  - The [YAML-defined rules](#creating-and-using-yaml-defined-rules-file) in a file.
+- Combination of [local rules with Semgrep Registry rules](#running-multiple-rules-simultaneously). 
 
-You can run all rules on your local code or continuously in your Source Code Management (SCM) service (such as GitHub or GitLab) with Semgrep in CI.
+You can run all rules on your code locally or continuously in your Source Code Management (SCM) service (such as GitHub or GitLab) with Semgrep in CI. For more information, see [Semgrep CI overview](semgrep-ci/overview.md).
 
-The following diagram displays the current number of Semgrep rules:
-<div className="lang-container" style={{marginBottom: '20px'}}>
-  <iframe width="900" height="400" frameBorder="0" src="https://dashboard.semgrep.dev/metric/semgrep-rules.num/graph"></iframe>
-</div>
-
-## Running Semgrep Registry rules
+## Running Semgrep Registry rules locally
 
-This section explains how to run [Semgrep Registry](https://semgrep.dev/explore) rules locally. See the following examples of how you may test Semgrep rules:
-
-1. Test how Semgrep automatically surveys languages and frameworks and run recommended rules for your source code:
-    ```sh
-    semgrep --config=auto path/to/src
-    ```
-2. Explore the [Semgrep Registry](https://semgrep.dev/explore) and choose a rule.
-3. On the page of the rule, click **Run Locally**.
-4. Copy the code for local install, and then add the path to the source code you want to check:
-    <pre class="language-bash"><code>semgrep --config="<span className="placeholder">RULESET-ID</span>" <span className="placeholder">PATH/TO/SRC</span></code></pre>
-5. Optional: Run registry rules simultaneously with local rules:
-   <pre class="language-bash"><code>semgrep --config="<span className="placeholder">RULESET-ID</span>" --config=<span className="placeholder">PATH/TO/YML PATH/TO/SRC</span></code></pre>
+Test how Semgrep automatically surveys languages and frameworks and run recommended rules for your source code (substitute the `PATH/TO/SRC` with path to your source code):
```

Comment:
```suggestion
You can test how Semgrep automatically surveys languages and frameworks and run recommended rules for your source code with the following command (substitute the `PATH/TO/SRC` with path to your source code):
```

Reply:
Giving this suggestion because as written, it sounds like a command "Test!".

Reply:
Alsooo, I think the (substitute the `PATH/TO/SRC` with path to your source code) is probably understandable? But @s-santillan can give her opinion on this.

Reply:
I think the question I have is, is this part exactly procedural? the part below it is. But this is leading you into it. You can also phrase it as "To test how semgrep....run:"